### PR TITLE
Backend/dev

### PR DESCRIPTION
**Pull request by tomdziwood**
Main point: changing keys of serialized fields.

### CODE REVIEW

File: src/main/java/pl/put/poznan/rulework/json/ClassificationSerializer.java [hunk 1]
```diff
@@ -33,54 +33,54 @@ public void serialize(Classification classification, JsonGenerator jsonGenerator
         jsonGenerator.writeFieldName("decisionsDomain");
         jsonGenerator.writeRawValue(mapper.writeValueAsString(classification.getOrderOfDecisions()));
 
-        jsonGenerator.writeFieldName("indicesOfCoveringRules");
+        jsonGenerator.writeFieldName("Indices of covering rules");
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/json/ClassificationSerializer.java:36]: This change is unnecessary

File: src/main/java/pl/put/poznan/rulework/json/CrossValidationSerializer.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public void serialize(CrossValidation crossValidation, JsonGenerator jsonGenerat
 
         jsonGenerator.writeStartObject();
 
-        jsonGenerator.writeFieldName("numberOfFolds");
+        jsonGenerator.writeFieldName("Number of folds");
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/json/CrossValidationSerializer.java:27]: This change is unnecessary

File: src/main/java/pl/put/poznan/rulework/json/RuleSetWithCharacteristicsSerializer.java [hunk 3]
```diff
@@ -38,7 +38,7 @@ public void serialize(RuleSetWithCharacteristics ruleSetWithCharacteristics, Jso
             jsonGenerator.writeRawValue(mapper.writeValueAsString(ruleCharacteristics));
 
             if(ruleCharacteristics instanceof ComputableRuleCharacteristics) {
-                jsonGenerator.writeFieldName("indicesOfCoveredObjects");
+                jsonGenerator.writeFieldName("Indices of covered objects");
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/json/RuleSetWithCharacteristicsSerializer.java:41]: This one too

**Comment by tomdziwood** [comment]
Is that all?

File: src/main/java/pl/put/poznan/rulework/model/RulesWithHttpParameters.java [hunk 4]
```diff
@@ -57,9 +57,9 @@ public void setTypeOfRule(RuleType typeOfRule) {
     public String toString() {
         return "RulesWithHttpParameters{" +
                 "ruleSet=" + ruleSet +
-                ", typeOfUnion=" + typeOfUnion +
+                ", typeOfUnions=" + typeOfUnions +
                 ", consistencyThreshold=" + consistencyThreshold +
-                ", typeOfRUle=" + typeOfRule +
+                ", typeOfRule=" + typeOfRule +
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/model/RulesWithHttpParameters.java:62]: Shouldn't it be 'typeOfRule**s**'?

Comment (tomdziwood) [reply src/main/java/pl/put/poznan/rulework/model/RulesWithHttpParameters.java:62]: that's right

File: src/main/java/pl/put/poznan/rulework/json/UnionWithSingleLimitingDecisionSerializer.java [hunk 5]
```diff
@@ -37,36 +37,36 @@ public void serialize(UnionWithSingleLimitingDecision unionWithSingleLimitingDec
...
-        jsonGenerator.writeFieldName("boundaryRegion");
+        jsonGenerator.writeFieldName("Boundary region");
         jsonGenerator.writeRawValue(mapper.writeValueAsString(unionWithSingleLimitingDecision.getBoundaryRegion()));
 
-        jsonGenerator.writeFieldName("objects");
+        jsonGenerator.writeFieldName("Objects");
         jsonGenerator.writeRawValue(mapper.writeValueAsString(unionWithSingleLimitingDecision.getObjects()));
 
-        jsonGenerator.writeFieldName("unionType");
+        jsonGenerator.writeFieldName("Union type");
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/json/UnionWithSingleLimitingDecisionSerializer.java:61]: This is unnecessary

File: src/main/java/pl/put/poznan/rulework/json/UnionWithSingleLimitingDecisionSerializer.java [hunk 6]
```diff
@@ -37,36 +37,36 @@ public void serialize(UnionWithSingleLimitingDecision unionWithSingleLimitingDec
...
         jsonGenerator.writeRawValue(mapper.writeValueAsString(unionWithSingleLimitingDecision.getUnionType()));
 
         serializeLimitingDecision(unionWithSingleLimitingDecision, jsonGenerator);
 
-        jsonGenerator.writeFieldName("accuracyOfApproximation");
+        jsonGenerator.writeFieldName("Accuracy of approximation");
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/json/UnionWithSingleLimitingDecisionSerializer.java:66]: This one too

File: src/main/java/pl/put/poznan/rulework/json/UnionWithSingleLimitingDecisionSerializer.java [hunk 7]
```diff
@@ -37,36 +37,36 @@ public void serialize(UnionWithSingleLimitingDecision unionWithSingleLimitingDec
...
         jsonGenerator.writeRawValue(mapper.writeValueAsString(unionWithSingleLimitingDecision.getAccuracyOfApproximation()));
 
-        jsonGenerator.writeFieldName("qualityOfApproximation");
+        jsonGenerator.writeFieldName("Quality of approximation");
```
Comment (dominieq) [inline src/main/java/pl/put/poznan/rulework/json/UnionWithSingleLimitingDecisionSerializer.java:69]: And this one as well

**Comment by tomdziwood** [comment]
Ok, i have changed locally all that codes line above. Is that all? Can I commit and merge?

**Pull request merged**